src/main/kotlin/com/exactpro/th2/dataprocessor/zephyr/cfg/CachesConfiguration.kt

```diff
@@ -28,4 +29,5 @@ class CachesConfiguration(
 class CacheConfiguration(
     val expireAfterSeconds: Long,
     val size: Int = 100,
+    val invalidateAt: LocalTime? = null,
```

Comment:
Maybe use a time at night as default value?

Reply:
I don't think that invalidating the whole cache by default is a good idea. This is why the default value is `null`